cypress/e2e/dashboard.cy.ts

```diff
@@ -65,4 +73,45 @@ describe('Dashboard', () => {
     cy.contains('42.00').should('not.exist')
     cy.contains('12.00')
   })
+
+  it('links to the envelope filtered transactions', function () {
+    cy.wrap(
+      Cypress.Promise.all([
+        createAccount({ name: 'Internal' }, this.budget),
+        createAccount({ name: 'External', external: true }, this.budget),
+      ]).then(([internalAccount, externalAccount]) => {
+        createTransaction(
+          {
+            sourceAccountId: internalAccount.id,
+            destinationAccountId: externalAccount.id,
+            envelopeId: this.firstEnvelope.id,
+            note: 'First Transaction',
+            amount: 1,
+          },
+          this.budget
+        )
+        createTransaction(
```

Comment:
We should add the two createTransaction calls to a Promise.all to make sure Cypress waits for them